### ISSUE TITLE
Is tinymce the same as Tiny Cloud?

### ISSUE THREAD

**Issue by sgon00** [tinymce/tinymce#4964]
Hi, sorry about this silly question. I am entirely confused the difference between tinymce and Tiny Cloud. If I use tinymce, do I have to follow Tiny Cloud Terms of Service?

Does the page https://www.tiny.cloud/pricing/ have anything to do with tinymce open source project?

Thanks a lot.

**Comment by alecpl** [comment]
This is the same. You can use TinyMCE from CDN (Cloud) or self-hosted. The basic functionality is for free. You can pay for "extras". You can also just get the source code and do what you want, it's LGPL licensed.

**Comment by sgon00** [comment]
@alecpl Thank you very much for the quick reply. I was wondering if I could remove "TINY Logo". From the [link](https://www.tiny.cloud/docs/general-configuration-guide/attribution-requirements/) is not very clear that if I can do it with the open source self-hosted version or not. 
I found another project which looked more open source yesterday and decided to try that one instead. So I don't really care of the logo question for now. Thank you very much again. Closing it.
